src/modules/auth/ui/views/__test__/sign-up-intergration.test.ts

```
@@ -0,0 +1,65 @@
+import { auth } from "@/lib/auth/auth";
+import { db } from "@/db";
+import { eq } from "drizzle-orm";
+import { account, user } from "@/db/schema";
+
+export const mockUser = {
+  body: {
+    email: "test@testing.com",
+    password: "12345678",
+    name: "John Doe",
+  },
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)